frontend/components/MainPage/sections/RewardsSection/index.tsx

```diff
@@ -38,34 +50,24 @@ const EarnedTagContainer = styled.div`
   top: -14px;
 `;
 
-const Loader = () => (
-  <Flex vertical gap={8}>
-    <Skeleton.Button active size="small" style={{ width: 92 }} />
-    <Skeleton.Button active size="small" style={{ width: 92 }} />
-  </Flex>
-);
-
 const DisplayRewards = () => {
   const {
     availableRewardsForEpochEth: reward,
     isEligibleForRewards,
     isStakingRewardsDetailsLoading,
-    isStakingRewardsDetailsError,
   } = useRewardContext();
-  const { isLoaded: isBalancesLoaded } = useBalanceContext();
-  const formattedReward =
-    reward === undefined ? NA : `~${balanceFormat(reward, 2)}`;
+  const { selectedStakingProgramId } = useStakingProgram();
 
-  const earnedTag = useMemo(() => {
-    if (isStakingRewardsDetailsLoading && !isStakingRewardsDetailsError) {
```

Comment:
I remember I've added `isStakingRewardsDetailsError` check due to some problems in the rewards call leaded to that earned badge was flickering. seems it's been resolved 🎉 hope you didn't see it too